tox.ini

```diff
@@ -0,0 +1,6 @@
+[tox]
+envlist = flake8
+
+[testenv:flake8]
+deps=flake8
+commands=flake8 --ignore=F403,E221,E501,F405 library
```

Comment:
@jaryn got it. I think the command should be: `commands=flake8 library/manageiq.py` ?
following your answer I'll merge this change and add a travis.yml.